pkg/config/engine/docker/docker.go

```diff
@@ -128,6 +129,28 @@ func (c Config) Save(path string) (int64, error) {
 		return 0, fmt.Errorf("unable to convert to JSON: %v", err)
 	}
 
-	n, err := engine.Config(path).Write(output)
+	n, err := config.Raw(path).Write(output)
 	return int64(n), err
 }
+
+// GetRuntime returns the TOML object of the runtime passed as input
+func (c *Config) GetRuntime(name string) (interface{}, error) {
+	if c == nil {
+		return nil, fmt.Errorf("config is nil")
+	}
+
+	config := *c
+
+	var runtimes map[string]interface{}
+	if _, exists := config["runtimes"]; exists {
+		runtimes = config["runtimes"].(map[string]interface{})
+	} else {
+		return nil, fmt.Errorf("config is nil")
```

Comment:
This is not true. This just means that no `runtimes` entry exists. What is the expected behaviour in this case?

Reply:
See my earlier response. I've changed this behaviour. No longer returning an `error`